### DISCUSSION TITLE
mm_sd15_v3.safetensors Image blur

### DISCUSSION THREAD

**Discussion by moqinchanyuan** [continue-revolution/sd-webui-animatediff#390] [General]
I downloaded the v3 model（mm_sd15_v3.safetensors），but the effect of v3 is much worse than that of the official v2. When can we support the official v3。

**Comment by moqinchanyuan** [comment]
web ui can run the v2(ckpt)

**Reply by Louis24** [reply]
mm_sd15_v3.safetensors this link, the gen images are still not gif
